docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -477,14 +477,18 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[aggMadScalars]
 [source, sql]
 --------------------------------------------------
 PERCENTILE(
-    field_name,  <1>
-    numeric_exp) <2>
+    field_name,         <1>
+    percentile[,        <2>
+    method[,            <3>
+    method_parameter]]) <4>
 --------------------------------------------------
 
 *Input*:
 
 <1> a numeric field
-<2> a numeric expression (must be a constant and not based on a field)
+<2> a numeric literal
+<3> optional string literal to indicate the <<search-aggregations-metrics-percentile-aggregation-approximation,percentile algorithm>>. Possible values: `tdigest` or `hdr`. Defaults to `tdigest`.
+<4> optional numeric literal for configuring the <<search-aggregations-metrics-percentile-aggregation-approximation,percentile algorithm>>. Configures `compression` for `tdigest` or `number_of_significant_value_digits` for `hdr`. The default is the same as that of the backing algorithm.
```

Comment:
Based on Andrei's review, `integer` may be a good replacement for `numeric literal`? I'd also streamline the wording slightly and remove the repeated link.
```suggestion
<4> optional integer that configures the percentile algorithm.
Configures `compression` for `tdigest` or `number_of_significant_value_digits`
for `hdr`. Uses the same defaults as the percentile algorithm.
```